torch/_library/custom_ops.py

```diff
@@ -662,6 +663,12 @@ def backend_select(keyset, *args, **kwargs):
     def __call__(self, *args, **kwargs):
         return self._opoverload(*args, **kwargs)
 
+    def register_vmap(
```

Comment:
Needs some tests for customopdef.register_vmap. As written I don't think this will work, customopdef needs to lazily register a function that can call self._vmap_fn to DispatchKey::FuncTorchBatched

Reply:
added